resources/keywords/dbaas_policy.resource

```diff
@@ -100,16 +103,91 @@ Click If Element Present
     END
 
 User Creates DBaaSPolicy For ${namespace} From Admin View As ${DBaaSPersona}
+    Set Test Variable   ${operatorNamespace}    ${namespace}
     User Navigates To DBaaSPolicy Screen    ${namespace}
-    Click If Element Present    ${lblOperand}
-    Wait Until Page Contains Element    ${btnCreateDBaaSPolicy}
-    Click Element    ${btnCreateDBaaSPolicy}
-    Create Custom DBaaSPolicy as ${DBaaSPersona}
+    User Creates DBaaSPolicy For ${DBaaSPersona}
 
 User Creates DBaaSPolicy For ${namespace} From Invalid Namespace Alert on Dev View As ${DBaaSPersona}
+    Set Test Variable   ${operatorNamespace}    ${namespace}
     User Navigates To Import MongoDB Provider Account On Developer View
     Wait Until Page Contains Element    ${errCreateDBaaSPolicy}          timeout=20s
     Click Element    ${errCreateDBaaSPolicy}
     Create Custom DBaaSPolicy as ${DBaaSPersona}
     User Selects Administrator View
 
+User Creates DBaaSPolicy For ${DBaaSPersona}
+    Click If Element Present    ${lblOperand}
+    Wait Until Page Contains Element    ${btnCreateDBaaSPolicy}
+    Click Element    ${btnCreateDBaaSPolicy}
+    Create Custom DBaaSPolicy as ${DBaaSPersona}
+
+Max Quota Exceeded Error Displays
+    Scroll Element Into View    ${txtQuotaError}
+    Element Should Be Visible   ${txtQuotaError}
+
+Specify Connection Namespaces And Labels
+    ${ns}=  Run Keyword And Return Status   Should Contain    ${nsAndLabel}    namespace
+    ${label}=  Run Keyword And Return Status   Should Contain    ${nsAndLabel}    label
+    IF      ${ns}
+        Click Element    ${btnToggleConnNS}
+        Add Policy Connection Namespaces    ${addNS}
+    END
+    IF    ${label}
+        Click Element    ${btnToggleConnNSSelect}
+        Element Should Be Visible    ${btnToggleMatchExp}
+        Click Element    ${btnToggleMatchExp}
+        Element Should Be Visible    ${btnAddMatchExp}
+        Add Policy Connection Nsselector    ${addLabel}
+    END
+
+User Creates DBaaSPolicy With ConnectionNamespace For ${namespace} As ${DBaaSPersona}
```

Comment:
Should we mention in which view the ConnectionNamespace is created?

Reply:
This action performed on DBaaSPolicy screen and irrespective of Openshift view, this would be same for all the flows. So keeping it general, I hope we should leave it as it is.

Reply:
But isn't consistent with the previous keywords.
Shouldn't it be consistent? Or maybe you mean that in this specific keyword only the view does not matter?

Reply:
Yes @ikanias For this keyword view does not matter.

Reply:
OK got it